django/db/backends/base/schema.py

```diff
@@ -197,6 +197,9 @@ def table_sql(self, model):
                 sql += ' ' + tablespace_sql
         return sql, params
 
+    def _column_default(self, field):
```

Comment:
Please rename to `_column_default_sql()` it sounds better to me, also a docstring would be helpful, e.g.
```
"""
Return the SQL to use in a DEFAULT clause. The resulting string should
contain a '%s' placeholder for a default value.
"""
```